Source/WebKit/_WebKit_SwiftUI/API/WebView.swift

```diff
@@ -36,13 +36,46 @@ public struct WebView: View {
     ///
     /// - Parameter page: The ``WebPage`` that should be associated with this ``WebView``. It is a programming error to create multiple ``WebView``s with the same ``WebPage``.
     public init(_ page: WebPage) {
-        self.page = page
+        self.storage = .webPage(page)
     }
 
-    private let page: WebPage
+    /// Create a new WebView with the specified URL.
+    ///
+    /// For example, you can create a WebView that displays one of two URLs depending on the state of a toggle:
+    ///
+    ///     struct URLView: View {
+    ///         @State private var url: URL? = nil
+    ///         @State private var toggle = false
+    ///
+    ///         var body: some View {
+    ///             VStack {
+    ///                 Button("Toggle") {
+    ///                     toggle.toggle()
+    ///                 }
+    ///                 WebView(url: url)
+    ///             }
+    ///             .onChange(of: toggle, initial: true) {
+    ///                 url = toggle ? URL(string: "https://www.webkit.org") : URL(string: "https://www.apple.com")
+    ///             }
+    ///         }
+    ///     }
+    ///
+    /// - Parameter url: The URL to display in the view. If this value is non-nil or changes to become a non-nil value, the new URL is loaded into the view.
+    public init(url: URL?) {
+        self.storage = .state(State(initialValue: WebPage()), url)
```

Comment:
Nit, but I’d avoid `init(initialValue:)`. It’s an old spelling that ended up being discarded from the `@propertyWrapper` API, but got left in for source compatibility. It also definitely eagerly invokes the initial value, which you’d want to avoid as much as possible (although I’m not actually sure who calls the `@autoclosure` version from the swiftinterface…).

```swift
  @available(iOS 17.0, macOS 14.0, tvOS 17.0, watchOS 10.0, *)
  @usableFromInline
  internal init(wrappedValue thunk: @autoclosure @escaping () -> Value) where Value : AnyObject, Value : Observation.Observable

  @_alwaysEmitIntoClient public init(initialValue value: Value) {
        _value = value
    }
```